ui/app/build-configs/detail/details-tab/pncBuildConfigDetailsTab.js

```diff
@@ -20,7 +20,8 @@
 
   angular.module('pnc.build-configs').component('pncBuildConfigDetailsTab', {
     bindings: {
-      buildConfig: '<'
+      buildConfig: '<',
+      productVersion: '='
```

Comment:
Hi Martin, the reason I am using the = is for updating the productVersion object after editing the BC, otherwise the changes to the BC will not be shown until user refresh the page manually.